core/common/src/DisplayStyleSettings.ts

```diff
@@ -706,6 +742,35 @@ export class DisplayStyleSettings {
     return this._modelAppearanceOverrides.size > 0;
   }
 
+  /** Set the planar clip mask for a persistent reality [[Model]] drawn by this display style.  Masking of BIM models is not supported although they can be used for masking for reality models and background maps.
+ * @param modelId The ID of the persistent reality [[Model]]
+ * @param planarClipMask The clip mask to apply to the [[Model]].
+ * @note If this style is associated with a [[ViewState]] attached to a [[Viewport]], use [[Viewport.overrideModelPlanarClipMask]] to ensure
+ * the changes are promptly visible on the screen.
+ * @see [[dropModelPlanarClipMaskOverride]]
+ * @beta
+ */
+  public overrideModelPlanarClipMask(modelId: Id64String, planarClipMask: PlanarClipMaskSettings): boolean { return this.changePlanarClipMaskOverride(modelId, true, planarClipMask); }
+
+  /** Remove planar clip mask applied to a [[Model]] by this style.
+   * @param modelId The ID of the [[Model]].

```

Comment:
`[[Thing]]` links only work if `Thing` is defined in the same package as the comment. Otherwise it produces broken links on the docs site. Use e.g., `[Model]($backend)` and `[Viewport]($frontend)`. otherwise.

Reply:
Fixed.